docs/learn-staking.md

```diff
@@ -70,78 +74,75 @@ The second point to note is that each validator candidate is free to name their
 
 ## Accounts
 
-There are three different accounts for managing your funds: `Stash`, `Controller` and `Session` accounts.
+There are two different accounts for managing your funds: `Stash` and `Controller`. The latter sets your [Session keys](learn-keys#session-keys).
 
-![staking](assets/NPoS/staking-keys.png)
+![staking](assets/NPoS/staking-keys_new.png)
 
-- **Stash:** This account holds funds bonded for participation, but delegates its staking and governance functions to controller and proxy keys. As a result, you may actively participate with a stash key kept in a cold wallet, meaning it stays offline all the time, possibly sharded in bank vaults. After unbonding, users must wait a certain amount of time in order to access the locked funds (600 blocks at the time of writing).
-- **Controller** This account controls its stash account's nomination of validator nodes, or authorizes operation of your own validator, switching between validating, nominating and idle. It only needs enough funds to post transactions when actions are taken.
-- **Proxy** This account participates in governance on behalf of its stash account. Again, it only needs enough funds to post vote transactions.
-- **Session** Session keys are not account keys, but instead consist of several different key types used by validator nodes for different functions. A validator operator first certifies their session keys with their controller key. We recommend handling session keys using only your node's RPC interface because if session keys exist elsewhere then you might equivocate and be slashed. We still support the legacy `--key` parameter for testnets like Alexander.
+* **Stash:** This account holds funds bonded for participation, but delegates its staking and governance functions to controller and proxy keys. As a result, you may actively participate with a stash key kept in a cold wallet, meaning it stays offline all the time. After unbonding, users must wait a certain amount of time in order to access the locked funds (600 blocks at the time of writing).
```

Comment:
- The unbonding period isn't really part of the definition of a Stash and should be covered elsewhere.
- Proxy is unrelated to staking but we should mention it and link (please verify my link works).
```suggestion
* **Stash:** This account holds funds bonded for staking, but delegates some functions to a Controller. As a result, you may actively participate with a Stash key kept in a cold wallet, meaning it stays offline all the time. You can also designate a Proxy account to vote in [governance](./learn-governance) proposals.
```